funnel/models/rsvp.py

```diff
@@ -46,12 +47,12 @@ def _space_rsvp_for(self, user, create=False):
 
 
 def _space_rsvps_with(self, status):
-    return self.rsvps.filter_by(status=status)
+    return self.rsvps.filter_by(_state=status)
 
 
 def _space_rsvp_counts(self):
-    return {value: count for value, count in db.session.query(Rsvp.status, db.func.count(Rsvp.status)).filter(
-        Rsvp.proposal_space == self).group_by(Rsvp.status)}
+    return dict(db.session.query(Rsvp._state, db.func.count(Rsvp._state)).filter(
+        Rsvp.proposal_space == self).group_by(Rsvp._state).all())
```

Comment:
This use of `_state` doesn't feel right, but I'll have to see what we can do about it.